Controllers/category.js

```diff
@@ -0,0 +1,23 @@
+const Category = require('../Models/category');
+
+exports.addCategory = async (req, res) => {
```

Comment:
just a suggestion we can export multiple functions with exports.module = {Category, getCategories}